rental.rb

```diff
@@ -0,0 +1,24 @@
+require 'date'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] I think it is more appropriate to use `attr_accessor` to define the `book and person` instance variables. By using `attr_accessor`, you can simplify the process of creating attributes in your classes while maintaining good encapsulation practices. 👍